src/app/browse-page/card-view/card-view.component.ts

```diff
@@ -89,7 +107,8 @@ export class CardViewComponent implements OnInit {
     this._router.navigate(['/browse','groups', this.browseLevel], { queryParams: queryParams })
   }
 
-  private search_owner(): void {
+  /** Implement the search of owner by owner_id */
+  private searchOwner(): void {
```

Comment:
Pass either the group or the owner_id to this function as an argument just to make it clear what this function relies on